### DISCUSSION TITLE
Isolation Forest / Autoencoder contamination parameter not effecting results?

### DISCUSSION THREAD

**Discussion by DrSouthmountain** [yzhao062/pyod#552] [Q&A]
Hello,

I have have run Isolation Forest and Autoencoder methods on my own datasets. I have tried to see what impact adjusting the contamination parameter makes on the resulting ROC and precision. As far as I can see nothing changes when adjusting the contamination parameter of the classifier from 0.1 to 0.5 (or anything in between).

I have tried the same with the isolation_forest example file from the repo (keeping the randomness stable by using the random_state parameter). 

Has anyone else run into the same? Or am I misunderstanding things?

**Comment by yzhao062** [comment]


**Reply by DrSouthmountain** [reply]
Ah I see! Thank you for your quick reply!

**Comment by yzhao062** [comment]
Contamination does not change roc which is based on raw scores. Contamination is only change threshold in to convert raw scores to binary anomaly labels.
